pkgs/development/python-modules/slither/default.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  prettytable,
+}:
+
+buildPythonPackage rec {
+  pname = "slither-analyzer";
+  version = "0.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10vrcqm371kqmf702xmqmzimv3xgrn3k3ip06nr1l6gnj3jk138g";
+  };
+
+  propagatedBuildInputs = [ prettytable ];
+
+  meta = with lib; {
+    description = "Static Analyzer for Solidity";
+    longDescription = ''
+      Slither is a Solidity static analysis framework written in Python 3. It
+      runs a suite of vulnerability detectors, prints visual information about
+      contract details, and provides an API to easily write custom analyses.
+    '';
+    homepage = https://github.com/trailofbits/slither;
+    license = licenses.agpl3;
+    maintainers = maintainers.asymmetric;
```

```suggestion
    maintainers =  [ maintainers.asymmetric ];
```
That's why the check is failing.